content/webapp/pages/series/[seriesId].tsx

```diff
@@ -79,12 +87,11 @@ export const getServerSideProps: GetServerSideProps<
   const client = createClient(context);
 
   // GOTCHA: This is for a series where we have the `webcomics` type.
+  // i.e.  the body squabbles series.
   // This will have to remain like this until we figure out how to migrate them.
   // We create new webcomics as an article with comic format, and add
   // an article-series to them.
-  const isWebcomics =
-    seriesQueryId === bodySquabblesSeriesId ||
-    seriesQueryId === 'body-squabbles';
```

Comment:
So I was going to say that this removes support for /series/[id] but I now realise that we never did and it was a duplicate? 😅 should we?? probably not if it's already unsupported and we want to be rid of it but oops.